mkdocs.yml

```diff
@@ -193,17 +195,8 @@ nav:
   - REST APIs:
     - Evaluation API: experiment/apis/evaluation-api.md
     - Management API (Beta): experiment/apis/management-api.md
-  # - Remove / Move:
-  #   - experiment/sdks/beta-migration.md
-  #   - Concepts:
-  #     - experiment/remove-move/concepts/index.md
-  #     - experiment/remove-move/concepts/experiment-user.md
-  #     - experiment/remove-move/concepts/experiment-sdk-configuration.md
-  #     - experiment/remove-move/concepts/experiment-variants.md
-  #   - Local Evaluation (Alpha):
-  #     - experiment/remove-move/local-evaluation/local-evaluation-overview.md
-  #     - experiment/remove-move/local-evaluation/create-a-local-evaluation-flag.md
-  #     - experiment/remove-move/local-evaluation/nodejs-sdk.md
+  - Notebooks:
+    - Sequential Testing vs T-Test: experiment/notebooks/sequential-testing-vs-t-test.ipynb
```

Comment:
Is there anything to keep in mind in terms of dependencies (like if we have a jupyter notebook that imports functions / classes from another jupyter notebook / python file) does that just work out of the box?